tests/python/frontend/tflite/test_forward.py

```diff
@@ -742,26 +742,35 @@ def _test_neg(data):
     """ One iteration of neg """
     return _test_unary_elemwise(math_ops.neg, data)
 #######################################################################
+# Square
+# ------
+
+def _test_square(data):
+    """ One iteration of square """
+    return _test_unary_elemwise(math_ops.square, data)
+#######################################################################
```

Comment:
> You could also refer to the implementation in `convert_squared_difference`, maybe could consider using `power`.

I implemented it with `multiply` since we already know how to do this in case the input is quantized. Later I saw that uint8 is still not supported... In general, I don't think multiply/power makes a lot of difference to the implementation.

Reply:
`power` will make us have chance to leverage the LLVM intrinsic of `pow` directly. This is one important different place. I prefer we use `power`. Like MXNet frontend, we convert `square` using `power` too. Please also refer `convert_squared_difference` of TFLite.

Reply:
Thanks, @wyc-ruiker @FrozenGene. I updated the PR with the recommendations.

Reply:
Here is a redundant line. I think we should delete it.